.github/workflows/ci_cd.yml

```diff
@@ -179,6 +179,7 @@ jobs:
       with:
         library-name: ${{ env.test-library-name }}
         use-trusted-publisher: true
+        skip-existing: true
```

Comment:
The default was ``true`` already, right?

Reply:
Good catch! Just enabled this input parameter.